omnigibson/transition_rules.py

```diff
@@ -971,7 +1004,10 @@ def _validate_nonrecipe_objects_not_contained(cls, recipe, in_volume):
         Returns:
             bool: True if none of the non-relevant objects are contained
         """
-        idxs = np.concatenate(cls._CATEGORY_IDXS[obj_category] for obj_category in recipe["input_objects"].keys())
+        if len(recipe["input_objects"]) == 0:
+            return True
```

Comment:
I don't think this is correct? Even if recipe["input_objects"] is empty, we still need to check no non-relevant objects are in volume?

Reply:
good point! I will fix this

Reply:
fixed